### DISCUSSION TITLE
[Wiki] An outdated link

### DISCUSSION THREAD

**Discussion by lewis-yeung** [ScoopInstaller/Scoop#5056] [General]
There is an outdated link in [this Wiki page](https://github.com/ScoopInstaller/Scoop/wiki/App-Manifests) after [a revision](https://github.com/ScoopInstaller/Scoop/wiki/Home/_compare/1ec10c47877a3e31a5f6a1372c290fe199f2d683#diff-355883cf07556dda17d052752e54eb44edf0104280e7f9c9f33fe04d4648bb17L24-R24):

> ... See [Pre- and post-install Scripts](https://github.com/ScoopInstaller/Scoop/wiki/Pre--and-Post-install-scripts) for more details.

The link should be "[Pre and Post install and uninstall scripts](https://github.com/ScoopInstaller/Scoop/wiki/Pre-and-Post-install-and-uninstall-scripts)".

**Comment by niheaven** [comment]
Fixed
